poll.go

```diff
@@ -83,7 +83,18 @@ func (h *Headscale) PollNetMapHandler(ctx *gin.Context) {
 		Str("machine", machine.Name).
 		Msg("Found machine in database")
 
-	machine.Name = req.Hostinfo.Hostname
+	hname, err := NormalizeNamespaceName(
```

Comment:
We should probably generalise the name of this function now.

Reply:
Perhaps make them two different functions? `NormalizeHostname` and `NormalizeNamespaceName`?

Reply:
I'm in favor of something generic since `hostname` and `namespace` are both part of a FQDN. So `NormalizeName` or `NormalizeDomainName` or `NormalizeHostname` should work too. 
I think I'll go with `NormalizeName` if that's ok with you.

Reply:
I think FQDN makes sense to have in the name, as that is what it is normalising for